src/Models/Webhook.php

```diff
@@ -79,6 +80,20 @@ public function tags()
         return Tag::whereIn('id', $this->tag_id)->get();
     }
 
+    public function appliedTags()
+    {
+        $tagsApplied = Tag::whereIn('id', $this->tag_id)->get();
+
+        return $tagsApplied->map(function ($tag) {
+            return $tag->name;
+        })->toArray();
```

Comment:
With eloquent it's always a better approach not to retrieve all entries at once. Especially on databases with millions of records this is going to be an issue. 

For these use cases you can use a chunking method, `each` for instance is once of those, but perhaps `map()` is also a method that chunks: like chunkMap: https://github.com/laravel/framework/blob/05d920410e58f04b13dcaa83937b9d4574fc9b3f/src/Illuminate/Database/Concerns/BuildsQueries.php#L81

Reply:
Thanks! What about the `chunk()` method? Did some reading, I could do something like this:

```
 $tagsApplied = [];

        Tag::whereIn('id', $this->tag_id)->chunk(100, function ($tags) use (&$tagsApplied) {
            $tagsApplied = array_merge($tagsApplied, $tags->map(function ($tag) {
                return $tag->name;
            })->toArray());
        });

 return $tagsApplied;
```

Reply:
If you're only after the name. I would use a select("name") and get(). You won't need to loop or chunk then.

Reply:
The initial one is still unnecessary:

```suggestion
        return Tag::select('name')->whereIn('id', $this->tag_id)->get()->toArray();
```

Is exactly the same thing.

Reply:
I tried the simplified version with select and pluck... Works on my end so I pushed the tweak. Have a look.